frontend/src/app/components/tasks-filters/tasks-filters.component.scss

```diff
@@ -0,0 +1,15 @@
+.task-actions-col {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.submitButton {
```

Comment:
Use the **kebab case** for CSS classes. Rename `submitButton` to `submit-button`.